frappe/patches/v8_0/update_gender_and_salutation.py

```diff
@@ -2,7 +2,7 @@
 
 from __future__ import unicode_literals
```

Comment:
Remove changes to version 8 patches and create a new patch file in v12 folder.

Reply:
Since this function has been split into multiple parts, we will need to fix these as well. This patch won't run again though, so it's a safe change.